userspace/libsinsp/event.cpp

```diff
@@ -2483,7 +2483,7 @@ const sinsp_evt_param* sinsp_evt::get_param_value_raw(const char* name)
 	{
 		if(strcmp(name, get_param_name(j)) == 0)
 		{
-			return &(m_params[j]);
+			return &(m_params.at(j));
```

Comment:
This is quite useless, as `get_param_name(j)` already validated `j` index for us. I'd stick with old version.